Development/src/components/CollapseButton.js

```diff
@@ -0,0 +1,15 @@
+import { IconButton } from '@material-ui/core';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+
+const CollapseButton = ({ onClick, isExpand }) => (
+    <IconButton
+        size="small"
+        title={isExpand ? 'Hide channels' : 'View channels'}
```

Comment:
Collapse Button da a generic name, but this is specific to channels. Maybe pull the title word out?